pgminer/Tuura/PGminer/Main.hs

```diff
@@ -35,7 +36,9 @@ main = do
                  Set.fromList [ (x, y) | x <- events, y <- events, x <= y ]
         co a b = cache Map.! (min a b, max a b)
         graphs = map (fmap decode) $ reduceLog co log
-        result = unlines . addIds $ map printGraphExpr graphs
+        result = if (optBool options /= "")
```

Comment:
Use pattern matching `case (optBool options)` here instead of `/= ""`, it usually leads to cleaner code.